dataflow/spanner-io/pom.xml

```diff
@@ -53,6 +53,7 @@
         <artifactId>maven-failsafe-plugin</artifactId>
         <version>2.19.1</version>
         <configuration>
+          <reportNameSuffix>sponge_log</reportNameSuffix>
```

Comment:
Should this be in java-repo-tools or somewhere else?

Reply:
The parent pom is a version behind, it probably just needs to be manually updated since we released a new version

Reply:
This is the only failsafe plugin usage in the whole project, so I wasn't sure if it was necessary to move it up. But the parent pom is the version where I added sponge_log to surefire - it's just that failsafe and surefire don't use the same plugin config.

Reply:
I ask questions when I review the answer can be, (and in this case appears to be), NO, this is the right spot.